Nos/Views/EditableText.swift

```diff
@@ -90,6 +90,15 @@ struct EditableText: UIViewRepresentable {
                 width = uiView.frame.size.width
                 Self.recalculateHeight(view: uiView, result: $calculatedHeight)
             }
+        } else if width == 0,
```

Comment:
This part was really tricky, but the text view wouldn't expand to fit the initial contents without it.